src/utils/ucc_proc_info.h

```diff
@@ -14,13 +14,18 @@ typedef uint32_t ucc_host_id_t;
 typedef uint32_t ucc_socket_id_t;
 
 typedef struct ucc_proc_info {
-    ucc_host_id_t   host_id;
+    ucc_host_id_t   host_hash;
     ucc_socket_id_t socket_id;
+    ucc_host_id_t   host_id;
```

Comment:
seems "host_id" is replaced with host_hash. do we nee need this ?

Reply:
host_id will be used in next PRs to build the subgroups for CL/HIER: host id is also a uniq number that identifies the host.  The difference from host_hash is that host_id \in [0, N-1], where N is the total number of nodes - ie., contiguous value - used to build subgroups.